tests/contrib/django_hosts/django_app/app_urls.py

```diff
@@ -1,8 +1,13 @@
-from django.conf.urls import url
+import django
 
 from .. import views
 
 
+if django.VERSION < (2, 0, 0):
+    from django.conf.urls import url
+else:
+    from django.urls import re_path as url
```

Comment:
same here, do we even test `django_hosts` with `django` 1?

Reply:
Nope min version is 2.2. Good catch